sql/bank_test.go

```diff
@@ -83,12 +83,9 @@ CREATE TABLE IF NOT EXISTS bank.accounts (
 	b.RunParallel(func(pb *testing.PB) {
 		for pb.Next() {
 			from := rand.Intn(*numAccounts)
-			var to int
-			for {
-				to = rand.Intn(*numAccounts)
-				if from != to {
-					break
-				}
+			to := rand.Intn(*numAccounts - 1)
+			if from == to {
+				to = *numAccounts - 1
```

Comment:
Are you sure? We decreased the domain of the `rand.Intn` to [0, *numAccounts - 1), so *numAccounts - 1 will only be used when `from == to`.